### PR TITLE
GitHub Actions: pin .NET SDK version

### PR DESCRIPTION
**Pull request by DedSec256**

### CODE REVIEW

**Review by auduchinok — APPROVED** [review]

**Comment by auduchinok** [comment]
@DedSec256 Could you also update global.json, so local builds use this version too, please?

**Comment by nojaf** [comment]
When you do update the `global.json` I think you don't want to allow newer versions `6.0.400` will have https://github.com/dotnet/fsharp/issues/13525 I believe.

**Comment by DedSec256** [comment]
@nojaf, this is why we are now fixing the sdk version on GH Actions (:

File: ReSharper.FSharp/global.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
   "sdk": {
+    "version": "6.0.301",
+    "rollForward": "patch",
     "allowPrerelease": false
   }
-}
+}
```
[the remark this answers is not in the document]
Comment (DedSec256) [reply ReSharper.FSharp/global.json:7]: Generated by Rider ¯\_(ツ)_/¯

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -34,6 +34,11 @@ jobs:
         working-directory: ./rider-fsharp
         run: ./gradlew prepare
 
+      - name: Set up .NET SDK
+        uses: actions/setup-dotnet@v1
```
Comment (nojaf) [inline .github/workflows/ci.yml:38]: If you use `v2` of this action you can actually point to your `global.json` file to set the version. 
See https://github.com/actions/setup-dotnet `global.json in a subdirectory:`

Comment (DedSec256) [reply .github/workflows/ci.yml:38]: Thanks for the tip, I'll try!

**Pull request merged**